src/main/java/com/ioet/bpm/people/config/DynamoDBConfig.java

```diff
@@ -47,6 +48,7 @@ public InitializingBean initializeTables(AmazonDynamoDB amazonDynamoDB) {
             List<Class> modelClasses = new ArrayList<>();
             modelClasses.add(Person.class);
             modelClasses.add(PasswordHistory.class);
+            modelClasses.add(UpdatePassword.class);
```

Comment:
This class won't be persisted in the database so it should not be included here.